docs/guide/development.md

```diff
@@ -0,0 +1,40 @@
+# App Mesh Controller Development Guide
+
+## Prerequisites
+
+* [Go 1.13+](https://golang.org/)
+* [Docker](https://www.docker.com/)
+* [Make](https://man7.org/linux/man-pages/man1/make.1.html)
+* [jq](https://stedolan.github.io/jq/)
+
+## Building the Controller
+
+The controller Docker image can be built via:
+
+```bash
+AWS_REGION=<region> AWS_ACCOUNT=<account ID> make docker-build
+```
+
+This command will compile the controller, run unit tests, and generate a Docker image using the ECR repository format (e.g. <account ID>.dkr.ecr.<region>.amazonaws.com/amazon/appmesh-controller).
```

Comment:
nit: <> won't be rendered correctly in markdown. You can either add quotes around  this `<account ID>.dkr.ecr.<region>.amazonaws.com/amazon/appmesh-controller` or change <>

Reply:
Fixed